app.js

```diff
@@ -28,6 +28,8 @@ var dust = require('dustjs-linkedin');
 var dustHelpers = require('dustjs-helpers');
 var cons = require('consolidate');
 
+const password_token = 'super-secret-f8ed84e8f41e4146403dd4a6bbcea5e418d24i7';
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-medium.svg" width="20"/></sub>&nbsp; Use of Hardcoded Credentials

Do not hardcode passwords in code. Found hardcoded password used in password_token.

*[CWE-798](https://cwe.mitre.org/data/definitions/798.html) | [CWE-259](https://cwe.mitre.org/data/definitions/259.html) | [Priority score 574](https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score) | Line 31*